recipe/meta.yaml

```diff
@@ -1,52 +1,72 @@
 {% set version = "2.1.3" %}
-{% set sha256 = "7c9ca6730638abcbd004909e7f4816b9e2c1486d3e864b1a26261ca84002cbf0" %}
+{% set number = 1 %}
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set scs_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
-  name: {{ name|lower }}
+  name: scs-split
   version: {{ version|replace('-', '.') }}    # 2.1.1-2 => 2.1.1.2
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/s/scs/scs-{{ version }}.tar.gz
+  sha256: 7c9ca6730638abcbd004909e7f4816b9e2c1486d3e864b1a26261ca84002cbf0
 
 build:
-  number: 0
-  script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py==27 and (win or (not x86_64))]
+  number: {{ number }}
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
-  host:
-    - libblas
-    - liblapack
-    - python
-    - setuptools
-    - pip
-    - numpy
-  run:
-    - python
-    - {{ pin_compatible('numpy') }}
-    - scipy >=0.13.2
 
-test:
-  imports:
-    - scs
-    - _scs_direct
-    - _scs_indirect
-  requires:
-    - nose
-  source_files:
-    - test/
-  commands:
-    - nosetests test
+outputs:
+  # A meta-package to select CPU or GPU build for scs.
+  - name: scs-proc
+    version: 1.0.0
+    build:
+      string: {{ scs_proc_type }}
+    test:
+      commands:
+        - exit 0
+
+  - name: scs
+    script: build-pkg.sh   # [not win]
+    script: build-pkg.bat  # [win]
+    build:
+      string: "py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ number }}_{{ scs_proc_type }}"                                                  # [cuda_compiler_version == "None"]
+      string: "py{{ CONDA_PY }}cuda{{ cuda_compiler_version|replace(".", "") }}h{{ PKG_HASH }}_{{ number }}_{{ scs_proc_type }}"  # [cuda_compiler_version != "None"]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - numpy                                  # [build_platform != target_platform]
+        - {{ compiler('c') }}
+        - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+      host:
+        - python
+        - setuptools
+        - pip
+        - numpy
+        - scipy
+        - libblas
+        - liblapack
+      run:
+        - python
+        - {{ pin_compatible('numpy') }}
+        - scipy
+      run_constrained:
+        - scs-proc =*={{ scs_proc_type }}
```

Comment:
```suggestion
        - scs-proc  * {{ scs_proc_type }}
```

Reply:
I intensely dislike the significant whitespace in this way of writing things; I'm aware it's not (yet? 😋) the standard in CF, but I'll keep using `=` for these things.